Lib/threading.py

```diff
@@ -1475,11 +1482,34 @@ def __init__(self):
         with _active_limbo_lock:
             _active[self._ident] = self
 
+# Helper thread-local instance to detect when a _DummyThread
+# is collected. Not a part of the public API.
+_thread_local_info = local()
+
+class __DeleteDummyThreadOnDel:
```

Comment:
```suggestion
class _DeleteDummyThreadOnDel:
```

Reply:
In the standard library, it is more common to use a single underscore prefix as the name for private classes.